### PR TITLE
Fix Habitat_Interactive_Tasks.py test due to refactoring of CameraSensor and SceneGraph

### PR DESCRIPTION
**Pull request by vauduong**
## Motivation and Context

See: https://github.com/facebookresearch/habitat-sim/pull/1036

Currently, there is a PR to embed RenderCamera to CameraSensor. This is a more user-friendly approach which allows each VisualSensor to render a scene with its own RenderCamera, rather than getting the default render camera from the SceneGraph. 

The PR deprecates SceneGraph::getDefaultRenderCamera(), which is invoked by the Habitat_Interactive_Tasks.py test in test_examples.py. Once PR #1036 is merged and habitat-sim is updated, this PR can be merged to fix Habitat_Interactive_Tasks.py.

## How Has This Been Tested

Build

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by mathfac** [comment]
Looks like it is blocked by HSIM changes now:
```
Traceback (most recent call last):
  File "/home/circleci/miniconda/envs/habitat/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/home/circleci/miniconda/envs/habitat/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/circleci/project/habitat-lab/test/test_examples.py", line 20, in run_main
    runpy.run_path(target, run_name="__main__")
  File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "examples/tutorials/nb_python/Habitat_Interactive_Tasks.py", line 755, in <module>
    sim, "rgb", crosshair_pos=[128, 190], max_distance=2.0
  File "examples/tutorials/nb_python/Habitat_Interactive_Tasks.py", line 707, in raycast
    sensor_name
AttributeError: 'habitat_sim._ext.habitat_sim_bindings.CameraSensor' object has no attribute 'get_render_camera'
```

**Comment by vauduong** [comment]
> Looks like it is blocked by HSIM changes now:
> 
> ```
> Traceback (most recent call last):
>   File "/home/circleci/miniconda/envs/habitat/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
>     self.run()
>   File "/home/circleci/miniconda/envs/habitat/lib/python3.6/multiprocessing/process.py", line 93, in run
>     self._target(*self._args, **self._kwargs)
>   File "/home/circleci/project/habitat-lab/test/test_examples.py", line 20, in run_main
>     runpy.run_path(target, run_name="__main__")
>   File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 263, in run_path
>     pkg_name=pkg_name, script_name=fname)
>   File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 96, in _run_module_code
>     mod_name, mod_spec, pkg_name, script_name)
>   File "/home/circleci/miniconda/envs/habitat/lib/python3.6/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "examples/tutorials/nb_python/Habitat_Interactive_Tasks.py", line 755, in <module>
>     sim, "rgb", crosshair_pos=[128, 190], max_distance=2.0
>   File "examples/tutorials/nb_python/Habitat_Interactive_Tasks.py", line 707, in raycast
>     sensor_name
> AttributeError: 'habitat_sim._ext.habitat_sim_bindings.CameraSensor' object has no attribute 'get_render_camera'
> ```

Thanks @mathfac , this change will be merged after HSIM changes are merged.

**Comment by erikwijmans** [comment]
Restarted CI now that this has landed in hab-sim.

**Review by Skylion007 — APPROVED** [review]

**Pull request merged**
